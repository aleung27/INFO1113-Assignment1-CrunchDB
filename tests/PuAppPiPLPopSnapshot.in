SET u 9 8 0 2
SET j 6 4 3
SNAPSHOT
POP u
POP j
LIST ENTRIES
SNAPSHOT
CHECKOUT 1
PLUCK u 1
PICK u 1
APPEND j 0 0
SNAPSHOT
ROLLBACK 2
LIST ENTRIES
PLUCK u 2
ROLLBACK 1
LIST ENTRIES
