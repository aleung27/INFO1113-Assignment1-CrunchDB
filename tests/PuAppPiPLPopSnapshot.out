> ok

> ok

> saved as snapshot 1

> 9

> 6

> j [4 3]
u [8 0 2]

> saved as snapshot 2

> ok

> 9

> 8

> ok

> saved as snapshot 3

> ok

> j [4 3]
u [8 0 2]

> 0

> ok

> j [6 4 3]
u [9 8 0 2]

